iio/iio.c

```diff
@@ -73,6 +73,9 @@
 #define IIOD_CONN_BUFFER_SIZE	0x1000
 #define NO_TRIGGER				(uint32_t)-1
 
+#define STRINGIFY(x) #x
```

Comment:
elegant solution but please move these 2 macros in no_os_util.h, they will be useful everywhere

Reply:
and obviously prefix them